corehq/apps/domainsync/management/commands/copy_domain.py

```diff
@@ -239,9 +243,20 @@ def run(self):
                           (doc["doc_type"], count, self.total, doc["doc_type"], doc["_id"])
                 else:
                     if not self.simulate:
-                        dt = DocumentTransform(doc, self.sourcedb, self.exclude_attachments)
-                        save(dt, self.targetdb)
+                        for i in reversed(range(5)):
+                            try:
+                                dt = DocumentTransform(doc, self.sourcedb, self.exclude_attachments)
+                                break
+                            except RequestError:
+                                if i == 0:
+                                    raise
+                        for i in reversed(range(5)):
+                            try:
+                                save(dt, self.targetdb)
+                            except (ResourceConflict, ParserError, TypeError):
```

Comment:
TypeError gets raised from here: 
    save(dt, self.targetdb)
    ... omitting full stack trace.
    TypeError: expected string or buffer

My best intuition was that this was data corruption somewhere in transition, because I only saw it once testing and otherwise that document was copied correctly.